src/helper/DrawerItems.tsx

```diff
@@ -1,173 +1,318 @@
 import * as React from 'react';
-import ListItemButton from '@mui/material/ListItemButton';
-import ListItemIcon from '@mui/material/ListItemIcon';
-import ListItemText from '@mui/material/ListItemText';
-import DashboardIcon from '@mui/icons-material/Dashboard';
-import PeopleIcon from '@mui/icons-material/People';
-import BarChartIcon from '@mui/icons-material/BarChart';
-import LayersIcon from '@mui/icons-material/Layers';
-import AssignmentIcon from '@mui/icons-material/Assignment';
-import Divider from '@mui/material/Divider';
-import { NavLink } from 'react-router-dom';
+import { ListItemButton, ListItemIcon, ListItemText, Box } from '@mui/material';
+import DashboardRoundedIcon from '@mui/icons-material/DashboardRounded';
+import GroupsRoundedIcon from '@mui/icons-material/GroupsRounded';
+import MoveToInboxRoundedIcon from '@mui/icons-material/MoveToInboxRounded';
+import BadgeRoundedIcon from '@mui/icons-material/BadgeRounded';
+import SettingsRoundedIcon from '@mui/icons-material/SettingsRounded';
+import AccountBoxRoundedIcon from '@mui/icons-material/AccountBoxRounded';
 import Collapse from '@mui/material/Collapse';
 import List from '@mui/material/List';
 import ExpandLess from '@mui/icons-material/ExpandLess';
 import ExpandMore from '@mui/icons-material/ExpandMore';
-import FileUploadIcon from '@mui/icons-material/FileUpload';
-import GridViewIcon from '@mui/icons-material/GridView';
+import FileUploadRoundedIcon from '@mui/icons-material/FileUploadRounded';
+import PersonAddRoundedIcon from '@mui/icons-material/PersonAddRounded';
+import LogoutRoundedIcon from '@mui/icons-material/LogoutRounded';
+import { NavLink } from 'react-router-dom';
+import { useState } from 'react';
 
 const linkStyle = {
   textDecoration: 'none',
-  color: '#4d4d4d',
+  color: '#808080',
 };
 
 export default function DrawerItems() {
-  const [open, setOpen] = React.useState(true);
+  const [open, setOpen] = useState(true);
+  const [selectedLink, setSelectedLink] = useState<string>('');
```

Comment:
I would add '/' as initial selected link so Dashboard is selected after loading for the first time. Otherwise there will be no page selected which could be confusing.

Reply:
That makes sense, thanks 👍🏻